cmd/fs-v1.go

```diff
@@ -1042,3 +1063,62 @@ func (fs *fsObjects) ListObjectsHeal(bucket, prefix, marker, delimiter string, m
 func (fs *fsObjects) ListBucketsHeal() ([]BucketInfo, error) {
 	return []BucketInfo{}, errors.Trace(NotImplemented{})
 }
+
+// SetBucketPolicy sets policy on bucket
+func (fs *fsObjects) SetBucketPolicy(bucket string, policy policy.BucketAccessPolicy) error {
+	return persistAndNotifyBucketPolicyChange(bucket, policyChange{false, policy}, fs)
```

Comment:
you can remove the function persistAndNotifyBucketPolicyChange and have the following code:
```
1068 func (fs *fsObjects) SetBucketPolicy(bucket string, policy policy.BucketAccessPolicy) error {      
1069         return writeBucketPolicy(bucket, fs, policy)                                               
1070 }                                                                                                  
```

DeleteBucketPolicy also needs similar change.

Note that you can remove `type policyChange struct{..}` from bucket-policy.go

Reply:
Also, we should update the fsObjects.bucketPolicies